README.md

```diff
@@ -17,8 +17,8 @@ Current features:
 
 Supported versions:
 
-*  Kubernetes 1.25-1.29
-*  OpenShift 4.11-4.14
+*  Kubernetes 1.26-1.30
```

Comment:
Without being able to test on 1.30 (no kind release available quite yet), I'm not sure we can say we support it...

Reply:
@naemono good point, do we just move it back until a Kind version is available ? I can redo this when that becomes available

Reply:
Yeah I would say just wait on this until we have a good version of it available and update then... Ensuring a full e2e run in this PR

Reply:
I updated with the image built by @barkbay, please take another look. TIA